docs/integrations/blueprism.mdx

```diff
@@ -0,0 +1,131 @@
+---
+sidebar_position: 2
+title: Blue Prism
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+# Unlock Document Parsing with Cradl AI in Blue Prism
+
+<br/>
+
+This guide will show you how to extract data from your documents with Cradl AI in Blue Prism.
+By the end of this guide you should be able to:
+ - Train your own AI model customized to your own documents.
+ - Set up a feedback loop so your AI model can train and get better for every document you process.
+ - Process your documents in your Blue Prism workflow using Cradl AI.
+
+All you need to get started is:
+- Blue Prism is installed on your computer.
+- Account on Cradl AI (If you don't have one yet, you can sign up [here](https://app.cradl.ai/login?redirect=signup&utm_campaign=docs).
+
+
+:::tip Scalable
+Cradl AI empowers you with the ability to create your own AI models. 15 examples are all you need to get started.
+:::
+
+<br/>
+
+## Overview
+
+<br/>
+The Cradl AI Blue Prism asset consists of these components:
+
+| Name          | Type | Description |
+|:------------| :------------- |:--- |
+| Cradl AI           | Visual Business Object (VBO) | This is where all the actions for this guide can be found. |
+| Cradl Demo Process | Process | A demo process that shows an example of how you can retrieve the parsed documents from your Flow in Cradl AI. |
+| Cradl AI Web API   | Web Api Service | The API itself. It is possible to make adjustments here for advanced cases that require customization, to get the most out of Cradl AI. |
+| Cradl File Server  | Web Api Service | To support all sorts of documents of different sizes, Cradl AI comes with a file server that is used behind the scenes. It is not recommended to make any changes in this component. |
+| Cradl Credentials  | OAuth 2.0 - Client Credentials | Make sure to add your own Client ID and Client Secret from Cradl Flow Builder in this object for everything to play along nicely. |
+
+<br/>
+
+## Step 1: Login to Cradl and setup a Flow
+
+<br/>
+
+There is very little setup required in Cradl before you are ready to add actions in your Blue Prism processes.
+All you have to do is:
+
+- Open the [Flow Builder](https://app.cradl.ai/flows) and press *create new*.
+- Select a pre-trained model to get started, you are free to customize the model to extract the fields you want after the first training. 
```

Comment:
It sounds a bit like you can only customize the mode after the first training. How about:
```suggestion
- Select a pre-trained model to get started, you are also free to customize the model to extract the fields you want. 
```